<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlaneRegistrationDialog</class>
 <widget class="QDialog" name="PlaneRegistrationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>830</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>打印平面标定</string>
  </property>
  <widget class="QPushButton" name="pb_OK">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>400</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_Cancel">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>400</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_Add">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>400</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>添加</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_Del">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>400</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_gen_translation">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>400</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>计算</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>511</width>
     <height>381</height>
    </rect>
   </property>
   <property name="title">
    <string>定标点采集</string>
   </property>
   <widget class="QTableWidget" name="table_widget_points">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>491</width>
      <height>351</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>x</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>y</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>z</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>rx</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>ry</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>rz</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>10</y>
     <width>271</width>
     <height>221</height>
    </rect>
   </property>
   <property name="title">
    <string>转换矩阵</string>
   </property>
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>251</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>240</y>
     <width>271</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>信息</string>
   </property>
   <widget class="QPlainTextEdit" name="planText_message">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>251</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pb_Clear">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>400</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>重置</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
